### ISSUE TITLE
can not login wandb at tutorial on colab

### ISSUE THREAD

**Issue by mucunwuxian** [ultralytics/yolov5#4723]
## 🐛 Bug

I could not login wandb at [tutorial on Google Colaboratory](https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb).

<br/>

## To Reproduce (REQUIRED)

Input:

```
# Weights & Biases  (optional)
%pip install -q wandb
import wandb
wandb.login()
```

Output:

The processing is frozen...

![image](https://user-images.githubusercontent.com/9916906/132665014-ce2e9329-0885-4443-91f6-360659012fb7.png)

After then, times out.

![image](https://user-images.githubusercontent.com/9916906/132665179-b217a3d9-7b3e-4551-9f1b-bdccd499258b.png)

<br/>

## Environment

- Google Colaboratory
- GPU: use (default setting)

<br/>

## Countermeasure

I could handle it with the following code.

```
# # Weights & Biases  (optional)
# %pip install -q wandb
# import wandb
# wandb.login()

!pip install wandb
!wandb login
import wandb

!wandb enabled
```

reference:
https://docs.wandb.ai/quickstart
https://github.com/wandb/client/issues/764#issuecomment-730023788

<br/>

## Additional context

I had you teach me that. :)
https://github.com/wandb/client/issues/764#issuecomment-729983871

ty!

<br/>

**Comment by morganmcg1** [comment]
Hmmm, I just tried this and couldn't reproduce...does it work if you click somewhere on the output cell after `wandb.login()` is called?

**Comment by glenn-jocher** [comment]
@mucunwuxian am also not able to reproduce (Chrome on MacOS).

**Comment by AyushExel** [comment]
@mucunwuxian is the issue still there after reloading the colab ?

**Comment by mucunwuxian** [comment]
Thank you for your kind cooperation. 
Oh, really...?

I and my coworker tried it on Chome on MacOS and Chome on WindowsOS.
But, I'm sorry that there was a difference in the procedure.

1. Open the [link](https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb).

2. Download as .ipynb file.

3. Upload to My Drive on Google Drive.

4. Execute uploaded .inpynb on Google Colab.

And now, I re-try thre above procedure from the beggining.
I am also not able to reproduce on Chrome on MacOS.
I am able to execute the `wandb.login()` without any problems.

But when execute with the file uploaded the other day, the same the problem happened...

![image](https://user-images.githubusercontent.com/9916906/132970930-41903e95-e5fb-4d27-ab4a-06c49e9aab8b.png)

↓

![image](https://user-images.githubusercontent.com/9916906/132970938-75830c8d-3de1-4f53-8368-e6ac99698c2d.png)

<br/>

However, after that, break execution, re-execute `wandb.login()`, successed.

![image](https://user-images.githubusercontent.com/9916906/132971177-6411d52e-48a6-4995-a48d-0127dec8ca63.png)

<br/>

In conclusion, the problem is gone.
I'm sorry to trouble you.

ty!🙇

<br/>

P.S.
Is it the similar issue...?
https://github.com/wandb/client/issues/2074

**Comment by glenn-jocher** [comment]
@mucunwuxian thanks for the info! We always recommend using the official YOLOv5 notebook, we can't speak to 3rd party notebooks naturally:
https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb
